en/docs/reference/data-formats/edi.md

```
@@ -4,7 +4,57 @@ title: EDI
 
 # EDI
 
-Electronic Data Interchange (EDI) is a standard format for exchanging business documents such as purchase orders, invoices, and shipping notices between organizations. The `ballerina/edi` module (v1.5.3) provides schema-driven, bidirectional conversion between EDI text and JSON.
+Electronic Data Interchange (EDI) is a set of standards that enables organizations to electronically exchange business documents such as purchase orders, invoices, and shipping notices in a structured, computer-readable format. Common EDI standards include X12 and UN/EDIFACT. The `ballerina/edi` module provides schema-driven, bidirectional conversion between EDI text and JSON.
+
+## EDI data format
+
+An EDI message is a sequence of **segments**. Each segment begins with a couple of character segment identifiers (e.g., `HDR`, `ITM`) followed by one or more **data elements** separated by a data element separator. A segment ends with a segment terminator. For example:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)